proto/ha_scope_config.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package dash.ha_scope_config;
+
+enum DesiredHaState {
+    HA_STATE_NONE = 0;
```

Comment:
Is `HA_STATE_NONE` a valid value for this field? If it is, please add `HA_STATE_UNSPECIFIED` as the first item.